testsuite/modern-testsuite/src/test/java/org/sonatype/nexus/testsuite/nuget/NugetProxyIT.java

```diff
@@ -132,7 +132,8 @@ public void sortOrderIsRespected() throws Exception {
 
     List<String> receivedHashes = extractPackageHashes(entries);
 
-    assertThat(receivedHashes, is(sorted(receivedHashes)));
+    Collator dictionaryOrder = Collator.getInstance(Locale.ENGLISH);
+    assertThat(receivedHashes, is(Ordering.from(dictionaryOrder).sortedCopy(receivedHashes)));
   }
 
 
```

Comment:
`@SuppressWarnings("unused")` would remove the compiler warning for `hashOrderedEntries` and make it clearer that it is intentional.